blocks/identity-block/components/SocialSignOn/index.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from 'react';
+import useIdentity from '../Identity';
+import './styles.scss';
+
+const SocialSignOn = ({ onError, redirectURL }) => {
+  const { Identity } = useIdentity();
+  const config = Identity?.configOptions ?? {};
+
+  const [isGoogleInitialized, setIsGoogleInitialized] = useState(false);
+  const [isFacebookInitialized, setIsFacebookInitialized] = useState(false);
+
+  if (!window.onFacebookSignOn) {
+    window.onFacebookSignOn = async () => {
+      try {
+        await Identity.facebookSignOn();
+        window.location = redirectURL;
+      } catch (e) {
+        onError();
+      }
+    };
+  }
+
+  useEffect(() => {
+    const fetchConfig = async () => {
+      await Identity.getConfig();
+    };
+    if (!Identity && !Identity.configOptions) {
```

Comment:
should be `if (Identity && !Identity.configOptions)`

Reply:
Wonderful